ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -110,7 +111,18 @@ public void onPageStarted(WebView webView, String url, Bitmap favicon) {
               SystemClock.nanoTime(),
               createWebViewEvent(webView, url)));
     }
-
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url){
+        ReactContext reactContext = (ReactContext) view.getContext();
+        if(url.startsWith("tel:")){
+            // handle by yourself
+            Intent intent = new Intent(Intent.ACTION_DIAL, Uri.parse(url)); 
```

Comment:
It'll probably suffice to just use a `Intent.ACTION_VIEW` for all URLs. In case of `tel:`, it won't dial the number, but will open the dialer with the number.